metrics.yaml

```diff
@@ -105,7 +105,7 @@ parameters:
 -
   description: 'Specify version for metrics components; e.g. for "openshift/origin-metrics-deployer:latest", set version "latest"'
   name: IMAGE_VERSION
-  value: "3.4.0"
+  value: "v3.5"
```

Comment:
Do we know this to be correct?  I had discussion with the various tags of our images and I recall '3.5.0' to be one. Just want to be consistent here.